packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { DetailsListBase } from './DetailsList.base';
 import { ISelection, SelectionMode, ISelectionZoneProps } from '../../utilities/selection/index';
 import { IRefObject, IBaseProps, IRenderFunction, IStyleFunctionOrObject } from '../../Utilities';
-import { IDragDropEvents, IDragDropContext } from './../../utilities/dragdrop/index';
+import { IDragDropEvents, IDragDropContext, IDragDropHelper, IDragDropOptions } from './../../utilities/dragdrop/index';
```

Comment:
Do we want a `docCategory` tag for these interfaces? And if yes, are they specific to DetailsList only? Same goes for the rest of the newly added API items to the surface.

Reply:
I defer to you as I'm less familiar with how these will / should be surfaced as part of our documentation

Reply:
If they are DetailsList specific than we should go with that. If not, then probably `Utilities` or maybe even leave them with no tags and they will be sorted out to `References`. @natalieethell we need another opinion 😄

Reply:
Though the drag & drop interfaces are named generically, they are fairly tightly coupled to `DetailsList`. So I think we should go with that category for now.

Reply:
Since these are used in both DetailsList and GroupedList, I'm thinking it might be best to give them their own pages under `References`.

In that case, we might want to label them with 

```tsx
/**
* {@docCategory IDragDropHelper}
*/
```

and 

```tsx
/**
* {@docCategory IDragDropOption}
*/
```

They need to have tags in order to be documented. I would say we shouldn't put them under `Utilities`, since `Utilities` just has components at the moment, not API reference pages.

Reply:
Clumped all of the drag & drop helper related interfaces under a single category as I find that most helpful. On their own the interfaces don't make much sense.